xdsl/ir/__init__.py

```diff
@@ -0,0 +1 @@
+from .core import *
```

Comment:
Does this still only import the things listed in the actual import? We still want to be able to import only Operation and not get everything else as well, right?